apps/mobile/package.json

```
@@ -3,19 +3,21 @@
-    "dev": "next dev --turbopack",
+    "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
     "@tanstack/react-query": "^5.71.0",
     "@tanstack/react-query-devtools": "^5.74.7",
+    "@toss/react": "^1.8.1",
     "@types/styled-components": "^5.1.26",
+    "next": "14.2.3",
```

Biome (1.9.4)

[error] The key next was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)